pinot-core/src/main/java/org/apache/pinot/core/geospatial/serde/GeometrySerializationType.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.geospatial.serde;
+
+import org.apache.pinot.core.geospatial.GeometryType;
+
+public enum GeometrySerializationType
+{
+    POINT(0, GeometryType.POINT),
+    MULTI_POINT(1, GeometryType.MULTI_POINT),
+    LINE_STRING(2, GeometryType.LINE_STRING),
+    MULTI_LINE_STRING(3, GeometryType.MULTI_LINE_STRING),
+    POLYGON(4, GeometryType.POLYGON),
+    MULTI_POLYGON(5, GeometryType.MULTI_POLYGON),
+    GEOMETRY_COLLECTION(6, GeometryType.GEOMETRY_COLLECTION),
+    ENVELOPE(7, GeometryType.POLYGON);
+
+    private final int id;
+    private final GeometryType geometryType;
+
+    GeometrySerializationType(int id, GeometryType geometryType)
+    {
+        this.id = id;
```

Comment:
Please use Pinot code style (name of member variables starts with `_` to avoid qualifying with `this`.